`          `**Title: Large-scale RNA-Seq Transcriptome Analysis of 4043 Cancers and 548 Normal TissueControls across 12 TCGA Cancer Types**

`  `***Author :**- A Study by Li Peng, Xiu Wu Bian, Di Kang Li, Chuan Xu,* 

*Guang Ming Wang, Qing You Xia, Qing Xiong.*
\*
`                                              `**Presented by Keshav Pande**

`                                               `**(<keshavpande1603@gmail.com>)**

![ref1]**Introduction** 

- RNA Seq technologies enable comprehensive analysis of gene expression in gene 
- The cancer genome atlas (TCGA) has accumulated data from more than 10,000 cancer patient and 25 + cancer types.
- The study aimsv to identify cross cancer and cancer specific gene signatures by analyzing differential expression data of over 4000 cancer samples and 548 normal tissues from the TCGA dataset.

![ref1]**Methodology**





![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.002.png)**1 ) Differential gene expression![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.003.png)**

**Fig.1** Heatmap of Differential gene express 







![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.004.png)![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.005.png)![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.006.png)   **2] Cro**

`   `**3] Cross-cancer gene Signature**        




![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.007.png)![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.008.png)






![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.009.png)


![ref2]**RESULTS** 

**1 ] Validation - Leave-One-Out Cross-Validation (LOOCV)** ![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.011.png)








![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.012.png)





4



![](Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.013.png)


`   `**2] Validation with Non-TCGA Datasets**

- **GSE40419**:  87 lung adenocarcinoma samples and 77 normal lung tissue samples with Predictive accuracy was 97.1%
- **GSE50760**: Colorectal cancer samples (18 primary cancer samples, 18 liver metastasis samples, and 18 normal colon samples). with Predictive accuracy was 96.23%

  ![ref2]**Conclusion: -** 

1. Cross-cancer gene signatures prove common molecular pathways, especially in cell cycle regulation, are disrupted across cancers.
1. Cancer-specific gene signatures confirm unique transcriptional alterations are key drivers in individual cancers.
1. The 14-gene signature’s high predictive accuracy proves its effectiveness in distinguishing cancerous from normal tissues.
1. Successful validation in non-TCGA datasets proves the gene signatures are robust and applicable across various data sources.
1. Both cross-cancer and cancer-specific signatures are proven valuable as diagnostic markers and therapeutic targets for personalized cancer treatment.

[ref1]: Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.001.png
[ref2]: Aspose.Words.661a75e0-649d-4993-8af8-d74121b424d2.010.png
